main.rb

```diff
@@ -73,6 +79,25 @@ class Saying
   end
 end
 
+put '/send-to-slack' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.